client/src/main/java/se/fortnox/reactivewizard/client/HttpClientConfig.java

```diff
@@ -190,4 +191,21 @@ public void setBasicAuth(String username, String password) {
             .setPassword(password);
     }
 
+    @Override
+    public boolean equals(Object object) {
```

Comment:
this might become a problem for classes extending this class. Why did you need this? Without this, you would create a new RxClientFactory for each config instance, but isn't there just a single instance anyway?

Reply:
My mistake.

Reply:
No, the configs will be a new instance for each resource.

Reply:
Perhaps we can change that by making this binding into a singleton? https://github.com/FortnoxAB/reactive-wizard/blob/master/config/src/main/java/se/fortnox/reactivewizard/config/ConfigAutoBindModule.java#L26

Reply:
Yes, that would have worked but now I went for a Class<? extends HttpClientConfig> as key instead.